src/components/ScrollableTabView/ScrollableTabView.tsx

```diff
@@ -77,34 +105,42 @@ class ScrollableTabView extends React.Component {
     });
   }
 
-  updateSelectedPage = (nextPage) => {
+  updateSelectedPage = (nextPage: number) => {
     let localNextPage = nextPage;
+
+    /**
+     * @TODO Check that block of code
+     */
     if (typeof localNextPage === 'object') {
-      localNextPage = nextPage.nativeEvent.position;
+      // @ts-ignore
+      localNextPage = nextPage?.nativeEvent?.position ?? 0;
     }
+    /**
+     * End
+     */
```

Comment:
Can we remove it?

Reply:
We need to check that part of code in future